Catalog/src/Domain/Entities/Product.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace Catalog.Domain.Entities;
+
+public class Product : BaseAuditableEntity

```

Comment:
I'd suggest having consistency in naming between requirements and implementation and between services.
In carting service and requirements this entity is named as Item.

Reply:
Fixed.